engine/src/main/java/org/entando/entando/aps/system/services/component/ComponentUsageEntity.java

```diff
@@ -26,7 +26,7 @@ public class ComponentUsageEntity implements Serializable {
     public static final String TYPE_CATEGORY = "category";
     public static final String TYPE_WIDGET = "widget";
     public static final String TYPE_FRAGMENT = "fragment";
-    public static final String TYPE_PAGE_TEMPLATE = "pageTemplate";
+    public static final String TYPE_PAGE_TEMPLATE = "pageModel";
```

Comment:
this creates confusion, change the constant name and possibly put a comment to describe why you did it (I know there is a ticket but usually when you read code you don't go to see git history and associated ticket)